packages/grid-pro/theme/lumo/vaadin-grid-pro-styles.js

```diff
@@ -46,6 +46,22 @@ registerStyles(
         var(--lumo-contrast-5pct) 14px
       );
     }
+
+    /* Loading editor cell styles are used by Flow GridPro */
+    :host([loading-editor]) [part~='focused-cell']::before {
+      content: '';
+      position: absolute;
+      inset: 0;
+      pointer-events: none;
+      box-shadow: inset 0 0 0 var(--_focus-ring-width) transparent;
+      animation: vaadin-grid-pro-loading-editor 1.4s infinite;
+    }
+
+    @keyframes vaadin-grid-pro-loading-editor {
+      50% {
+        box-shadow: inset 0 0 0 var(--_focus-ring-width) var(--_focus-ring-color);
```

Comment:
Animating opacity would be more performant. Animating the box-shadow requires a paint cycle, whereas opacity animation could be done purely by the GPU compositor.

Reply:
Sorry for the late review. I’ll make a PR to update the animation.

Reply:
https://github.com/vaadin/web-components/pull/8203

Reply:
This approach was copied from date-picker Lumo styles: https://github.com/vaadin/web-components/blob/256ef92ffd4bd5128b6362738779dad4419f4b7a/packages/date-picker/theme/lumo/vaadin-month-calendar-styles.js#L116